eea/reports/browser/report.pt

```diff
@@ -495,7 +495,7 @@
             </tal:file>
           </li>
 
-          <li tal:condition="python:item.portal_type in ['Document', 'SOERMessage', 'SOERKeyFact']">
+          <li tal:condition="python:item.portal_type in ['Document', 'Fiche']">
```

Comment:
@alecghica  I would get rid of the many portal_type checks and have only two:
One for non folderish content types which will get the normal fields and one for the folderish items which
we can go inside them to also display their children